components/concourse-operator/pkg/apis/concourse/v1beta1/pipeline_types.go

```diff
@@ -23,14 +23,22 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type PipelineConfig struct {
+	atc.Config
+}
```

Comment:
minor thing: not sure this has to be a struct, I think you could do this instead:
```suggestion
type PipelineConfig atc.Config
```
This would get rid of some of the `foo.Config.Config.bar` stuff below.  I don't feel strongly about this so feel free to ignore.

Reply:
I considered doing this ... but:

* While this would allow removing `foo.Config.Config` if would require doing some `atc.Config(foo.Config)` since it would be a different type
* Go's type system is weird enough that I'm not 100% sure if that would mean I have overidden any methods that might be on atc.Config

... the Config.Config way is ugly, but I understand it bit better :)